napari/_qt/_qapp_model/qactions/_view.py

```diff
@@ -74,6 +74,19 @@ def _zoom_out(viewer: Viewer):
     viewer.camera.zoom /= 1.5
 
 
+def _toggle_canvas_ndim(viewer: ViewerModel):
+    """Toggle the current canvas between 3D and 2D."""
+    if viewer.dims.ndisplay == 2:
+        viewer.dims.ndisplay = 3
+    else:  # == 3
+        viewer.dims.ndisplay = 2
+
+
+def _current_view_is_3d(viewer: Viewer):
```

Comment:
I'm not seeing where this is used? I think it's a leftover, can we remove?

Reply:
yeah this was from when the toggle was on. Will push a removal now.

Reply:
Done